db/migrate/20150421161752_add_disabled_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisabledToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :disabled, :boolean
```

Comment:
The [gds-sso README says](https://github.com/alphagov/gds-sso/blob/master/README.md#integration-with-a-rails-3-app) that this column should default to `false`.